prow/tide/search.go

```diff
@@ -40,7 +40,7 @@ func floor(t time.Time) time.Time {
 	return t
 }
 
-func search(query querier, log *logrus.Entry, q string, start, end time.Time, org string) ([]PullRequest, error) {
+func search(query querier, log *logrus.Entry, q string, start, end time.Time, org string) ([]CodeReviewCommon, error) {
```

Comment:
Since this is a GitHub specific search function I would have expected this to return a slice of PullRequests and for that slice to be converted separately, but if this is a more convenient place to do the change that seems fine.